app/workers/fast_track/hypertension_medication_request_data.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module FastTrack
+  class HypertensionMedicationRequestData
```

Comment:
Could we move this and the other non-job classes (below) into another directory that's not app/workers (my first thought is app/services)?

Reply:
@ericboehs I'm happy to do that. The reason I kept everything together for the moment is that this is a pilot and the plan is to delete this code if it doesn't move forward. I was just feeling like keeping it all together would make that easier but I can move all the helper classes if that's the preference.

Reply:
Do you feel like that is blocking or could I follow up this pr with one on Monday that pulls these non-job classes out?

Reply:
I think if they're name spaced as FastTrack and in fast_track directory (within app/services), it won't be hard to find them and remove.

I don't think moving them to app/services should affect any code so it should be very simple to do.

Reply:
Sure - I'm not worried about it being too much work :-) I'll move the files once all the other comments are addressed. I don't want to mundge the rest of the conversation on this pr.

Reply:
Addressed here: e447857781b9f1efd4af0dc4dcbb25e0c203b28d